controllers/articles.controller.js

```diff
@@ -13,7 +13,11 @@ exports.getArticleById = (request, response, next) => {
 }
 
 exports.getArticles = (request, response, next) => {
-    fetchArticles()
+    const queryKeys = Object.keys(request.query)
+    const invalidKeys = queryKeys.filter((key) => key !== 'sort_by' && key !== 'order')
+    const sort_by = invalidKeys.length ? "invalid_query" : request.query.sort_by
```

Comment:
It's not clear to me what this is for. If your endpoint is passed queries that aren't handled, we can just ignore them. When you extract the queries you want from req.queries (like you've done on the line below this comment!), we implicitly disregard any other queries.

Reply:
Thank you, I see your point. I have sorted this out now!